src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/BamboraClientImpl.java

```diff
@@ -123,7 +128,7 @@ public URL calculateSinglePaymentURL(String correlationId, BamboraTransType tran
 			paramString = paramString + this.hashKey;
 	
 			 //TODO - Complete logging once available.
-			//logger.info("Calculating MD5 on paramString " + paramString);
+			logger.info("Calculating MD5 on paramString " + paramString);
```

Comment:
is there any PI in this string, unless necessary, don't see the value of logging all the parameters

Reply:
@shaunmillargov your thoughts here? I think the comment was originally committed by you.

Reply:
Agree with @alexjoybc. No need to include this particular output. I had placed it there more for debugging.

Reply:
removed this line from code